.gitignore

```diff
@@ -20,4 +20,4 @@
 *.rar
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
-hs_err_pid*
+hs_err_pid*
```

Comment:
Because you just copied this file without understanding your calculator.jar didn't add to the repository. Please find out why it happened. And add a blank new line to the end of the file.